src/solver/simulation/include/antares/solver/simulation/solver.hxx

```diff
@@ -153,15 +151,7 @@ public:
             double* randomReservoirLevel = nullptr;
 
             // 1 - Applying random levels for current year
-            if (hydroHotStart && firstSetParallelWithAPerformedYearWasRun)
-            {
-                randomReservoirLevel = state[numSpace]
-                                         .problemeHebdo->previousYearFinalLevels.data();
```

Comment:
Is `previousYearFinalLevels` still used ? If not, please remove its definition from headers.

Reply:
it's already removed